followthemoney/property.py

```diff
@@ -85,8 +87,11 @@ def __init__(self, schema: "Schema", name: str, data: PropertySpec) -> None:
         self._label = data.get("label", name)
         self._description = data.get("description")
 
+        #: This property is deprecated and should not be used.
+        self.deprecated = as_bool(data.get("deprecated", False))
+
         #: This property should not be shown or mentioned in the user interface.
-        self.hidden = as_bool(data.get("hidden", False))
+        self.hidden = as_bool(data.get("hidden", self.deprecated))
```

Comment:
I’m not sure whether it’s a good idea to make this decision in FtM, maybe it’s better to leave that to the application?

Haven’t thought about how exactly we’d want to handle deprecated schemata/properties in Aleph, but one option would be to display existing entities using the deprecated schema, but disallow users from creating new entities.

Reply:
Removed.